hybrid/dispatch/power_storage/simple_battery_dispatch_heuristic_baseload.py

```diff
@@ -0,0 +1,149 @@
+import pyomo.environ as pyomo
+from pyomo.environ import units as u
+
+import PySAM.BatteryStateful as BatteryModel
+import PySAM.Singleowner as Singleowner
+
+from hybrid.dispatch.power_storage.simple_battery_dispatch import SimpleBatteryDispatch
+
+
+class SimpleBatteryBaseloadDispatchHeuristic(SimpleBatteryDispatch):
```

Comment:
This class should inherit the `SimpleBatteryDispatchHeuristic` class the `__init__` methods are exactly the same.